src/frontend/sharedComponents/ProjectInviteBottomSheet/InviteSuccessBottomSheetContent.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {BottomSheetContent} from '../BottomSheetModal';
+import {defineMessages, useIntl} from 'react-intl';
+import GreenCheck from '../../images/GreenCheck.svg';
+import {useNavigationFromRoot} from '../../hooks/useNavigationWithTypes';
+import {CommonActions} from '@react-navigation/native';
+
+const m = defineMessages({
+  goToMap: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.goToMap',
+    defaultMessage: 'Go To Map',
+  },
+  success: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.success',
+    defaultMessage: 'Success',
+  },
+  youHaveJoined: {
+    id: 'sharedComponents.ProjectInviteBottomSheet.youHaveJoined',
+    defaultMessage: 'You have joined {projName}',
```

Comment:
same nit about using `projectName` as the placeholder name

Reply:
https://github.com/digidem/comapeo-mobile/pull/366/commits/a5e25113e5b74d9b925c9b6f8fc79639206a0e4e